rero_ils/accounts_views.py

```diff
@@ -46,6 +53,25 @@ def user_exists(email):
         raise ValidationError(_('INVALID_USER_OR_PASSWORD'))
 
 
+def validate_password(password):
+    """Validate the password."""
+    length = current_app.config.get('RERO_ILS_PASSWORD_MIN_LENGTH', 8)
+    special_char = current_app.config.get('RERO_ILS_PASSWORD_SPECIAL_CHAR')
+    if len(password) < length:
+        raise ValidationError(
+            _('Field must be at least %(min)d characters long.', min=length))
```

Comment:
As password length check is already include into `password_validator`, why this test here ?